c7n/tags.py

```diff
@@ -591,6 +600,10 @@ class TagDelayedAction(Action):
             - type: mark-for-op
               op: stop
     """
+    deprecations = (
+        deprecated.optional_fields(('hours', 'days'), '2021-06-30'),
+        deprecated.optional_field('tag', '2021-06-30'),
```

Comment:
I'm not sure if we actually want to deprecate the optionality of the tag field. Explicit is better than implicit IMO.

Reply:
we're not deprecating tag fwiw